app/pages/verificationwithpassword/verificationwithpassword.js

```diff
@@ -58,6 +59,12 @@ export let VerificationWithPassword = React.createClass({
     this.setState({ loading: false });
   },
 
+  componentDidMount: function() {
+    if (this.props.trackMetric) {
+      this.props.trackMetric('VCA Home Verification - Screen Displayed');
```

Comment:
I suppose in blip we don't have the strings of the metrics separated out as constants? Can you double-check?